galasa-managers-parent/galasa-managers-testingtools-parent/dev.galasa.sdv.manager.ivt/build.gradle

```diff
@@ -0,0 +1,30 @@
+plugins {
+    id 'galasa.manager.ivt'
+}
+
+description = 'Galasa SDV Manager IVT'
+
+version = '0.34.0'
+
+dependencies {
+    implementation project (':galasa-managers-cicsts-parent:dev.galasa.cicsts.ceci.manager')
+    implementation project (':galasa-managers-core-parent:dev.galasa.core.manager')
+    implementation project (':galasa-managers-cicsts-parent:dev.galasa.cicsts.manager')
+    implementation project (':galasa-managers-zos-parent:dev.galasa.zos3270.manager')
+    implementation project (':galasa-managers-zos-parent:dev.galasa.zos.manager')
+    implementation project (':galasa-managers-core-parent:dev.galasa.artifact.manager')
+    implementation project (':galasa-managers-testingtools-parent:dev.galasa.sdv.manager')
+}
+
+// Note: These values are consumed by the parent build process
+// They indicate which packages of functionality this OSGi bundle should be delivered inside,
+// or referenced from.
+// The settings here are gathered together by the build process to create a release.yaml file 
+// which gathers-up all the packaging metadata about all the OSGi bundles in this component.
+ext.projectName=project.name
+ext.includeInOBR          = false
```

Comment:
You probably want to set the InOBR flag, so it gets included with all the other managers.